### DISCUSSION TITLE
Update guide for RC1

### DISCUSSION THREAD

**Discussion by alkampfergit** [microsoft/semantic-kernel#4002] [Q&A]
I've updated reference to RC1, basically everything seems to be broken, there is no anymore an IKernel interface, BasicPlanner does not work because the Microsoft.semantickernel.planners.core is still in beta and expect the IKernel, the RunAsync method of kernel seems to be gone.

I was not able to find a comprehensive guide that contains all the breaking changes for the RC1, especially the SequentialPlanner problema makes impossible to upgrade.

![image](https://github.com/microsoft/semantic-kernel/assets/358545/87a446bf-fcf1-44a9-a199-e5510fd67538)

**Comment by andrzejb75** [comment]
I have the same impression, it's very difficult to fix the broken parts which popped up after the upgrade to RC2. All existing documentation and code samples are relevant to beta version, not the RC2.

**Comment by andrzejb75** [comment]
> I've updated reference to RC1, basically everything seems to be broken, there is no anymore an IKernel interface, BasicPlanner does not work because the Microsoft.semantickernel.planners.core is still in beta and expect the IKernel, the RunAsync method of kernel seems to be gone.
> 
> I was not able to find a comprehensive guide that contains all the breaking changes for the RC1, especially the SequentialPlanner problema makes impossible to upgrade.
> 
> ![image](https://private-user-images.githubusercontent.com/358545/288150420-87a446bf-fcf1-44a9-a199-e5510fd67538.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MjA3ODksIm5iZiI6MTcwMTkyMDQ4OSwicGF0aCI6Ii8zNTg1NDUvMjg4MTUwNDIwLTg3YTQ0NmJmLWZjZjEtNDRhOS1hMTk5LWU1NTEwZmQ2NzUzOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwN1QwMzQxMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYzQzMTc0NzI3NGRlNmI2YTFlYWUxNzIxYzk5NjRjNmI4OGY3ODY4YjM5NzQ5MDYwZTI4ZTE4OGE4YzljOWE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.DJDAvDM5YKvZQPnpkV3aiWQxH0-FNcgAr6ltZk9StYE)

I think this is what we both have been looking for https://devblogs.microsoft.com/semantic-kernel/introducing-the-v1-0-0-beta1-for-the-net-semantic-kernel-sdk/

**Reply by Cotspheer** [reply]
Thats related to beta1 to beta8 but not beta8 to rc1 or rc3. For example IKernel was removed as far as I see, Microsoft.SemanticKernel.TemplateEngine.Basic that is mentiond in this guide as "added" was not promoted to rc3 so either it got renamed, replaced or removed. Sadly not the guide we are looking for I guess.

**Comment by alkampfergit** [comment]
This is the updated blog post. It seems that planners are not active part
of the library anymore


https://devblogs.microsoft.com/semantic-kernel/release-candidate-1-for-the-semantic-kernel-net-sdk-is-now-live/

Il gio 7 dic 2023, 06:11 Thomas Erni ***@***.***> ha scritto:

**Comment by evchaki** [comment]
The documentation has been updated here - https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel?view=semantic-kernel-dotnet and here is a blog post about some of the updates - https://devblogs.microsoft.com/semantic-kernel/semantic-kernel-v1-0-1-has-arrived-to-help-you-build-agents/ and https://devblogs.microsoft.com/semantic-kernel/migrating-from-the-sequential-and-stepwise-planners-to-the-new-handlebars-and-stepwise-planner/
